sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -191,7 +192,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
                 val actual = unindentAndTrim(
                   hiveResultString(df.queryExecution.executedPlan).mkString("\n"))
                 val expected = unindentAndTrim(output)
-                assert(actual === expected)
+                assert(actual.sorted === expected.sorted)
```

Comment:
Why do you sort them? After the changes, output of failed tests is unusable. Instead of `"[true]" did not equal "[false]"` I am getting `"[ertu]" did not equal "[aefls]"`

Reply:
From my understanding, output of examples must be precisely the same as in the examples. Let me try to remove the sorting.

Reply:
oh I thought `actual` and `expected` is a list of results. Sorry didn't catch it during review. @MaxGekk can you help to fix it? thanks!

Reply:
@cloud-fan Here is the PR https://github.com/apache/spark/pull/27324 , please, review it.